WinUIGallery/ControlPages/ItemsViewPage.xaml

```diff
@@ -0,0 +1,311 @@
+<!--
+//*********************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+// THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF
+// ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY
+// IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR
+// PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.
+//
+//*********************************************************
+-->
+<local:ItemsPageBase
+    x:Class="AppUIBasics.ControlPages.ItemsViewPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:AppUIBasics"
+    xmlns:localcp="using:AppUIBasics.ControlPages">
+
+    <local:ItemsPageBase.Resources>
+        <DataTemplate x:Key="ImageTemplate" x:DataType="localcp:CustomDataObject">
```

Comment:
Is it worth adding below templates to the xaml sample code section?

Reply:
Added templates to ControlExample.Xaml sections.